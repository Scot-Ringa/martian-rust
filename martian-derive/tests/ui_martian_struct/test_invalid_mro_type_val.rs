use martian_derive::MartianStruct;

struct Foo(i32);

#[derive(MartianStruct)]
struct InvalidField {
    num_reads: i64,
    #[mro_type=int] // Should be "int"
    foo: Foo,
}

fn main() {}
