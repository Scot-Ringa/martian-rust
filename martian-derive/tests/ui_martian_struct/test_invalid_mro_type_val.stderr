error: unexpected token: `int`
 --> $DIR/test_invalid_mro_type_val.rs:8:16
  |
8 |     #[mro_type=int] // Should be "int"
  |                ^^^
